lib/helpers/deployments.rb

```diff
@@ -30,12 +36,24 @@ def deploy(branch, deleted)
   end
 
   def deploy_module(module_name)
-    command = if settings.use_mcollective
-                "#{settings.command_prefix} mco r10k deploy_module #{module_name} #{settings.mco_arguments}"
-              else
-                "#{settings.command_prefix} r10k deploy module #{module_name}"
-              end
-    message = run_command(command)
+    # Currently requires puppet_webhook to be run as a non-root
+    # user with access to running the MCollective client.
+    if settings.use_mcollective
+      result = PuppetWebhook::Mcollective.new('r10k',
+                                              'deploy',
+                                              {
+                                                dtimeoute: settings.discovery_timeout,
```

Comment:
cut&paste, or is it really spelled like that? (if so, maybe a comment that the spelling is intentional)

Reply:
Good catch. I just spelled that wrong.